<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battle</class>
 <widget class="QWidget" name="Battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>对战</string>
  </property>
  <widget class="QWidget" name="CentralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>640</height>
    </rect>
   </property>
   <widget class="QPushButton" name="BtnStart">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>74</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent no-repeat;
background-image:url(:/images/res/startButton.png);
}
QPushButton:hover{
background:transparent no-repeat;
background-image:url(:/images/res/startButton_h.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnBackStep">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>74</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent no-repeat;
background-image:url(:/images/res/backStepButton.png);
}
QPushButton:hover{
background:transparent no-repeat;
background-image:url(:/images/res/backStepButton_h.png)
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnDraw">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>500</y>
      <width>74</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent no-repeat;
background-image:url(:/images/res/drawButton.png);
}
QPushButton:hover{
background:transparent no-repeat;
background-image:url(:/images/res/drawButton_h.png);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnFail">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>500</y>
      <width>75</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent no-repeat;
background-image:url(:/images/res/failButton.png);
}
QPushButton:hover{
background:transparent no-repeat;
background-image:url(:/images/res/failButton_h.png);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTotalTime">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCurChess">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>230</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTotalChessNum">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
